80s Game/Assets/Scripts/TargetManager.cs

```diff
@@ -153,7 +153,7 @@ void SpawnTarget(int targetIndex)
         target.isOnScreen = true;
         target.currentState = TargetStates.Moving;
         target.GetComponent<AnimationHandler>().ResetAnimation();
-        target.GetComponent<CircleCollider2D>().isTrigger = false;
+        target.GetComponent<PolygonCollider2D>().isTrigger = false;
```

Comment:
I'll need to readd this in the code for my PR. A lot of the bat logic got moved to a separate class for the behavior refactor